### PR TITLE
docs: Add Consul 1.10 release notes

### PR DESCRIPTION
**Pull request by blake**

### CODE REVIEW

File: website/content/docs/release-notes/1-10-0.mdx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+- **Streaming Enabled by Default for Service Health:** Streaming is a major architectural enhancement in how update notifications for blocking queries are delivered within the cluster which significantly reduces CPU and network bandwidth usage for large-scale Consul deployments. In Consul 1.10, streaming is now available for the service health HTTP endpoint and is enabled by default.
+
+- **Redesigned UI and Observability Enhancements:** The Consul UI has been redesigned with a new sidebar layout and dark theme. Additionally, Kubernetes users now have the ability to deploy Prometheus via the Consul Helm chart which automatically integrates it with Consul’s Service Visualization UI for displaying traffic metrics between services. Lastly, Pod and Envoy metrics can now be exposed to Prometheus using Kubernetes annotations via a single aggregated endpoint.
```
Comment (mikemorris) [inline website/content/docs/release-notes/1-10-0.mdx:16]:
```suggestion
- **Redesigned UI and Observability Enhancements:** The Consul UI has been redesigned with a new sidebar layout. Additionally, Kubernetes users now have the ability to deploy Prometheus via the Consul Helm chart which automatically integrates it with Consul’s Service Visualization UI for displaying traffic metrics between services. Lastly, Pod and Envoy metrics can now be exposed to Prometheus using Kubernetes annotations via a single aggregated endpoint.
```
```suggestion
- **Redesigned UI and Observability Enhancements:** The Consul UI has been redesigned with a new sidebar layout and dark theme. Additionally, Kubernetes users now have the ability to deploy Prometheus via the Consul Helm chart which automatically integrates it with Consul’s Service Visualization UI for displaying traffic metrics between services. Lastly, Pod and Envoy metrics can now be exposed to Prometheus using Kubernetes annotations via a single aggregated endpoint.
```
```suggestion
- **Redesigned UI and Observability Enhancements:** The Consul UI has been redesigned with a new sidebar layout. Additionally, Kubernetes users now have the ability to deploy Prometheus via the Consul Helm chart which automatically integrates it with Consul’s Service Visualization UI for displaying traffic metrics between services. Lastly, Pod and Envoy metrics can now be exposed to Prometheus using Kubernetes annotations via a single aggregated endpoint.
```

File: website/content/docs/release-notes/1-10-0.mdx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+- **Deprecation of Legacy ACL System:** In Consul 1.4, we upgraded to a new Access Controls (ACLs) system. This upgrade made improvements in Consul’s ACL system handles the API, Tokens, and Policies. With Consul 1.10, the legacy system is officially deprecated, and will be removed from Consul in a later release. We strongly recommend that users begin the process of migrating to the newer ACL system.
+
+## What's Changed
+
+- Drops support for Envoy version 1.13.x.
+- (Enterprise Only) Consul Enterprise has removed support for temporary licensing. All server agents must have a valid license at startup and client agents must have a license at startup or be able to retrieve one from the servers.
+
+For more detailed information, please refer to the [1.10.0](https://github.com/hashicorp/consul/releases/tag/v1.10.0) release page.
```
Comment (mikemorris) [inline website/content/docs/release-notes/1-10-0.mdx:25]:
```suggestion
For more detailed information, please refer to the [upgrade details page](https://www.consul.io/docs/upgrading/upgrade-specific#consul-1-10-0) and the [1.10.0 changelog](https://github.com/hashicorp/consul/releases/tag/v1.10.0).
```

**Review by mikemorris — CHANGES_REQUESTED** [review]
Made two suggestions, removing "dark theme" language, and linking to the upgrade details page, which in turn links to the FAQ and detailed upgrade instructions for more information on the impact of licensing changes.

**Review by david-yu — APPROVED** [review]
LGTM

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/400731.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 8a832635b8c6789694c81c8423332a76bbb6fadc onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 8a832635b8c6789694c81c8423332a76bbb6fadc onto `release/1.10.x` succeeded!

**Pull request merged**